### DISCUSSION TITLE
Question regarding the recent potentially device-bricking bug

### DISCUSSION THREAD

**Discussion by abceleung** [dangowrt/owrt-ubi-installer#173] [Q&A] [answered]
Hi, 

It seems like there is a wave of bricked RT3200/E8450 as seen in the [OpenWrt forum thread](https://forum.openwrt.org/t/belkin-rt3200-linksys-e8450-wifi-ax-discussion/94302/4353?u=abceleung), and the developers are working hard to fix this. Thanks for your effort!

I have a E8450 with OpenWrt 23.05.2 installed and is still functioning properly. Is my device at risk of bricking? Do I need to do anything now to prevent my device from bricking? 

(I think someone in the forum thread mentioned that setting performance governor helps)

**Comment by huyz** [comment]
See https://forum.openwrt.org/t/belkin-rt3200-linksys-e8450-wifi-ax-discussion/94302/4297

**Reply by dangowrt** [reply, marked as answer]
The cause for OKD has been found and it turns out only installer v1.0.3 or later were affected:
https://openwrt.org/toh/linksys/e8450#check_if_your_device_is_at_risk_of_okd

If you are affected, follow the steps to avoid OKD when running 23.05.x release here:
https://openwrt.org/toh/linksys/e8450#recovery_from_live_openwrt_2203x_or_2305x_with_fip_in_mtd_layout

**Reply by mikehardy** [reply]
This is just a thanks for the work discovering+fixing the OKD issue, and making the installers for this device.
I have 3, one suffered OKD and the other 2 I got lucky and they kept running.
I was able to unbrick the OKD'd one with a USB-Serial dongle along with the instructions on the OpenWRT page, and now I've got all 3 devices up and running (hopefully for many many years!) on the new OpenWRT release.
Fewer devices in the landfill has to be a good thing, I know I'm pleased.
Cheers
